string name
BoundingBox hook_box
Mask hook_mask
float32 conf_hook
BoundingBox tip_box
Mask tip_mask
float32 conf_tip
BoundingBox lowpoint_box
Mask lowpoint_mask
float32 conf_lowpoint